src/java.desktop/macosx/native/libawt_lwawt/awt/a11y/CommonComponentAccessibility.m

```diff
@@ -56,6 +56,7 @@ + (void) initializeRolesMap {
     [rolesMap setObject:@"StaticTextAccessibility" forKey:@"label"];
     [rolesMap setObject:@"RadiobuttonAccessibility" forKey:@"radiobutton"];
     [rolesMap setObject:@"CheckboxAccessibility" forKey:@"checkbox"];
+    [rolesMap setObject:@"SliderAccessibility" forKey:@"slider"];
```

Comment:
Shouldn't you also increase the rolesMap initial capacity?

Reply:
> Shouldn't you also increase the rolesMap initial capacity?

ok, I will do that in next commit.

Reply:
Done